src/aspnet-buzz/app/components/eventStream.ts

```diff
@@ -0,0 +1,113 @@
+/// <reference path="../../typings/react/react.d.ts"/>
+/// <reference path="../../node_modules/typed-react/typed-react.d.ts"/>
+/// <reference path="../../typings/jquery/jquery.d.ts"/>
+/// <reference path="../../typings/signalr/signalr.d.ts"/>
+
+import React = require("react");
+import TypedReact = require("typed-react");
+import Models = require("./GitHub/GitHubModels");
+import GitHub = require("./GitHub/GitHubEvents");
+var supportedEvents: string[] = [
+    "IssuesEvent",
+    "PullRequestEvent",
+    "IssueCommentEvent",
+    "PullRequestReviewCommentEvent",
+    "PushEvent",
+    "CreateEvent",
+    "DeleteEvent",
+    "WatchEvent",
+    "ForkEvent"
+];
+
+var issuesEvent = {"Type":"IssuesEvent","ID":2736226780,"Repo":{"Name":"aspnet/dnvm"},"Payload":{"Action":"closed","Number":0,"Issue":{"Html_Url":"https://github.com/aspnet/dnvm/issues/218","Title":"Beta4 DNVM points to the myget aspnetrelease feed instead of nuget.org","User":{"Login":"muratg","Avatar_Url":"https://avatars.githubusercontent.com/u/1808428?v=3"},"Number":218},"Comment":null,"Commits":null,"Pull_Request":null,"Ref":null,"Ref_Type":null},"Actor":{"Login":"muratg","Avatar_Url":"https://avatars.githubusercontent.com/u/1808428?"}};
```

Comment:
Yep
________________________________
From: David Fowler<mailto:notifications@github.com>
Sent: ‎26/‎04/‎2015 16:45
To: davidfowl/aspnet-buzz<mailto:aspnet-buzz@noreply.github.com>
Cc: Henrik Andersson<mailto:henrik.a.andersson@outlook.com>
Subject: Re: [aspnet-buzz] [WIP] Move to ReactJS (#1)

> +import TypedReact = require("typed-react");
> +import Models = require("./GitHub/GitHubModels");
> +import GitHub = require("./GitHub/GitHubEvents");
> +var supportedEvents: string[] = [
> +    "IssuesEvent",
> +    "PullRequestEvent",
> +    "IssueCommentEvent",
> +    "PullRequestReviewCommentEvent",
> +    "PushEvent",
> +    "CreateEvent",
> +    "DeleteEvent",
> +    "WatchEvent",
> +    "ForkEvent"
> +];
> +
> +var issuesEvent = {"Type":"IssuesEvent","ID":2736226780,"Repo":{"Name":"aspnet/dnvm"},"Payload":{"Action":"closed","Number":0,"Issue":{"Html_Url":"https://github.com/aspnet/dnvm/issues/218","Title":"Beta4 DNVM points to the myget aspnetrelease feed instead of nuget.org","User":{"Login":"muratg","Avatar_Url":"https://avatars.githubusercontent.com/u/1808428?v=3"},"Number":218},"Comment":null,"Commits":null,"Pull_Request":null,"Ref":null,"Ref_Type":null},"Actor":{"Login":"muratg","Avatar_Url":"https://avatars.githubusercontent.com/u/1808428?"}};

Test events?

---
Reply to this email directly or view it on GitHub:
https://github.com/davidfowl/aspnet-buzz/pull/1/files#r29107221